llama-index-integrations/vector_stores/llama-index-vector-stores-oracledb/llama_index/vector_stores/oracledb/base.py

```diff
@@ -95,25 +96,23 @@ def _escape_str(value: str) -> str:
 column_config: Dict = {
     "id": {"type": "VARCHAR2(64) PRIMARY KEY", "extract_func": lambda x: x.node_id},
     "doc_id": {"type": "VARCHAR2(64)", "extract_func": lambda x: x.ref_doc_id},
-    "text": {
-        "type": "CLOB",
-        "extract_func": lambda x: _escape_str(
-            x.get_content(metadata_mode=MetadataMode.NONE) or ""
-        ),
+    "embedding": {
+        "type": "VECTOR",
+        "extract_func": lambda x: array.array("f", x.get_embedding()),
     },
     "node_info": {
-        # Now specifying the column as CLOB intended for JSON, with a check constraint
         "type": "JSON",
         "extract_func": lambda x: json.dumps(x.node_info),
     },
     "metadata": {
-        # Also specified as CLOB intended for JSON, with a check constraint
         "type": "JSON",
         "extract_func": lambda x: json.dumps(x.metadata),
     },
-    "embedding": {
-        "type": "VECTOR",
-        "extract_func": lambda x: _stringify_list(x.get_embedding()),
+    "text": {
+        "type": "CLOB",
+        "extract_func": lambda x: _escape_str(
+            x.get_content(metadata_mode=MetadataMode.NONE) or ""
+        ),
```

Comment:
Unable to insert nodes containing a lot of text. [From Oracle documentation](https://docs.oracle.com/en/error-help/db/ora-24816/?r=23ai), `CLOB` binds should be at the end of the bind list. This is fixed by moving the text column to the last position of the declared global column_config dict.